docs/enterprise-solutions/configuration/remote-configuration/litellm/member-configuration.mdx

```diff
@@ -0,0 +1,168 @@
+3. The base URL should already be configured by your administrator
+4. Click **Save** to store your credentials
+
+<Tip>
+API keys are stored locally in VS Code and are only used by the Cline extension.
+</Tip>
+</Accordion>
+
+<Accordion title="Open Access (No Authentication)">
+If your LiteLLM proxy is configured for open access within your network:
+
+1. Select or confirm the **LiteLLM** provider is selected  
+2. Leave the API key field empty
+3. The extension will connect directly to the configured proxy endpoint
+4. No additional authentication is required
+
+<Info>
+Open access is common when the LiteLLM proxy is deployed within a secure network environment.
+</Info>
+</Accordion>
+
+<Accordion title="Custom Configuration">
+If your organization uses custom authentication or specific connection parameters:
+
+1. Follow any custom instructions provided by your administrator
+2. Contact your IT team if you encounter connection issues
+3. Additional configuration may be needed outside of VS Code
+
+<Note>
+Custom configurations might require specific network settings or additional authentication steps.
+</Note>
+</Accordion>
+</AccordionGroup>
+</Step>
+
+<Step title="Select Available Models">
+Once connected, you'll see the models available through your organization's LiteLLM proxy:
+
+- View available models in the model dropdown
+- Models are determined by your administrator's proxy configuration  
+- You can switch between models for different types of tasks
+- Some models may be restricted based on your access level
+
+<Tip>
+**Model Selection**
+
+Choose models based on your task requirements:
+- **Fast models** (like GPT-3.5-turbo) for quick responses
+- **Powerful models** (like GPT-4) for complex reasoning
+- **Specialized models** for code generation or specific domains
+</Tip>
+</Step>
+
+<Step title="Test the Connection">
+Send a test message in Cline to verify your connection works correctly with the LiteLLM proxy.
+
+<Tip>
+**Testing Recommendation**
+
+Test the connection in plan mode first to verify everything works correctly before using it for actual development tasks.
+</Tip>
+</Step>
+</Steps>
+
+## Model Usage
+
+### Available Model Categories
+The models available through your LiteLLM proxy typically include:
+
+**Text Generation Models:**
+- OpenAI GPT-4, GPT-3.5-turbo variants
+- Anthropic Claude 3 Sonnet, Haiku, Opus
+- Open source models like Llama 2, Mistral
+
+**Code-Specific Models:**
+- OpenAI GPT-4 for code
+- CodeLlama variants
+- Specialized code completion models
+
+**Multimodal Models:**
+- GPT-4 Vision for image analysis
+- Claude 3 models with vision capabilities
+
+### Model Selection Strategy
+Choose models based on your development needs:
+
+- **Quick iterations**: Use faster, cost-effective models
+- **Complex problems**: Use more powerful models  
+- **Code-heavy tasks**: Use code-specialized models
+- **Visual content**: Use multimodal models when working with images
```

Comment:
```suggestion
```
Let's remove this section since it will be harder to maintain as new models come out.